react-components/src/utilities/queryKeys.ts

```diff
@@ -7,6 +7,7 @@ export const queryKeys = {
   all: ['cdf'] as const,
   // ASSETS
   assetsById: (ids: IdEither[]) => [...assets, ids] as const,
+  assetsList: (filter?: any) => [...queryKeys.all, 'assets', 'list', filter],
```

Comment:
Are we using this queryKey?
If not, do we need to add this now for future usage?

Reply:
Overhead code 😄